src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -43,11 +45,22 @@ interface IResultsViewOncoprintProps {
     addOnBecomeVisibleListener?:(callback:()=>void)=>void;
 }
 
+export enum SortByUrlParamValue {
+    CASE_ID = "case_id",
+    CASE_LIST = "case_list",
+    DATA = "data",
+    DATA_MUTATION = "data_mut",
```

Comment:
rename as @jjgao recommended?

Reply:
All references to `data` and related sort subtypes were removed. Renaming is not relevant for this code anymore.